language/move-prover/bytecode-to-boogie/test_mvir/verify-stdlib/vector.mvir

```diff
@@ -0,0 +1,132 @@
+// A variable-sized container that can hold both unrestricted types and resources.
+
+// This hasn't been specified, yet.  It's here to satisfy a dependency with
```

Comment:
Specification here will not be needed and in fact be ignored. vector is only needed for mvir type checking, the boogie translator will ignore this module and instead expect the definitions (including specs if needed) in the prelude. Perhaps change comment to: "This is only needed for MVIR type check, as the definitions for Vectors are included in the prelude."

Reply:
Done